recipes/bed-reader/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "bed-reader" %}
+{% set version = "0.2.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  # Can't use the standard pattern because of the hypen/underscore remapping.
+  url: https://files.pythonhosted.org/packages/source/b/bed-reader/bed_reader-{{ version }}.tar.gz
```

Comment:
Can you do `https://pypi.io/packages/source/b/bed-reader/bed_reader-{{ version }}.tar.gz` instead? https://pypi.io/ will redirect to files.pythonhosted.org but the former seems to be more stable than the latter.

Reply:
Done! Thanks for taking a look @ocefpaf